### DISCUSSION TITLE
RingDAO Radical Token Economic Reform Research

### DISCUSSION THREAD

**Discussion by hackfisher** [ringecosystem/discussion#4] [General]
**Background**:  
RingDAO’s treasury income has experienced slow growth, while continuous RING issuance and liquidity outflows have significantly pressured the token price, market-making, and liquidity. This has eroded confidence among investors and traders. The current treasury and incentive models have not effectively responded to market demand or supported ecosystem growth as expected.

It's important to note that projects like Polkadot are also exploring revisions to their inflation models to adapt to market changes. This discussion serves as a research and discussion tool to explore restructuring RingDAO’s token economy in response to market conditions and RING’s current tokenomics, ensuring the long-term sustainability of the ecosystem.

---

### Reform Plan:

1. **Token Issuance Mechanism Modification**  
   - **Cancel New Issuance**:  
     Cease the issuance of new RING tokens to reduce supply pressure, create scarcity, alleviate the Treasury deficit, stabilize supply-demand dynamics, and ease long-term liquidity and market-making pressures.

2. **Darwinia Chain Incentive Mechanism Adjustment and Treasury Support**  
   - **Treasury Allocation and Long-Term Incentive Plan**:  
     Allocate RingDAO’s treasury to establish a long-term on-chain incentive plan to support the continuous operation of Darwinia Chain. This ensures transparent fund usage, fiscal stability, and sustained user engagement. The plan is inspired by the fiscal model of GnosisDAO and Gnosis Chain.  
   - **Staking Rewards Adjustment**:  
     Optimize the Kton and Collator staking rewards model (e.g., adopting a halving schedule every four years) to balance incentives with sustainable operations. Over time, increased on-chain activity could provide income for Collator nodes (similar to Sequencers).

3. **Budget Control Measures**  
   - **Strict Control of R&D and Marketing Budgets**:  
     Implement strict controls over R&D and marketing budgets until treasury income significantly improves. Resources must be efficiently allocated, and expenditures minimized. R&D and marketing efforts should focus on projects that align with RingDAO’s goals, ensuring product-market fit before scaling up. Community promotion efforts can still occur, leveraging strategies akin to MemeCoin viral marketing.

---

### Expected Effects of the Reform:

- **Stabilized Token Price**: By canceling new issuance, supply pressure will be reduced, contributing to price stabilization.
- **Sustainable Long-Term Growth**: Treasury allocation and the establishment of a permanent on-chain incentive plan will ensure the ecosystem's long-term sustainability.
- **Controlled Inflation Risks**: Adjusting staking rewards will mitigate inflation risks while maintaining network security.
- **Enhanced Resource Allocation**: Budget control will reduce unnecessary spending, freeing up resources for market-making and ecosystem growth.

---

### Treasury Allocation and Long-Term Chain/Module Incentive Plan

With the Treasury token supply fixed under the new token economy model, the focus shifts to how RingDAO will allocate these limited resources to various modules and ecosystem investments. The budget must be distributed according to the actual needs of each module to maximize the effective use of funds.

- **Long-Term On-Chain Incentive Adjustment for Darwinia Chain**:  
  A long-term on-chain incentive plan will be established for Darwinia Chain, ensuring continued on-chain activity and user engagement. The plan will be dynamically adjusted based on on-chain activities and ecosystem growth to address changing market demands.

- **Budget Allocation Tailored to Development Needs**:  
  RingDAO will reserve sufficient budget for other modules and ecosystem investments, adjusting these allocations based on the future development needs of each module. This approach ensures that all parts of the ecosystem receive the necessary financial support to avoid resource waste or shortfalls.

By drawing from the fiscal models of GnosisDAO and Gnosis Chain, this approach ensures that even with limited fiscal resources, RingDAO can effectively support long-term development across its ecosystem.

---

### Discussion Points:

- The impact of these measures on existing token holders and participants, and their acceptability.
- The timeline and transition plan for implementing the reform.
- Ensuring transparency and building community consensus throughout the reform process. Future proposals related to this solution are expected to leverage RingDAO's governance framework.

---

This research aims to fundamentally restructure RingDAO’s token economy, ensuring long-term sustainability. We invite feedback and suggestions for further refinement.

**Comment by SasoLithops** [comment]
Very interesting reform @hackfisher.

If we do that, do we cancel new issuance all together?

What if RING DAO decides how much new Ring is minted to support growth, pay developers, reward stakers... Not to have constant issuance, but issuance on demand for example every 6 months. A proper research, discussion and proposal should be given to RING DAO members for ammount of Ring to be minted each 6 months in advance. Ring that stays unused from previous half year gets burned.  

The impact is impossible to avoid.  Our staking rewards remained to high for too long. It is hard to trust a token that has that high double digit staking rewards in 2024. It has to lower down and price of token should guide the value of rewards. I have said this many times. I would rather have 5 Ring rewards and Ring being 0,1USD then 500 RING and Ring being 0,00001 USD. Sadly most crypto holders are in huge loss and are only trying to protect what is left of their bags by talking trash and offer no solution and just nag if one is given. I understand their frustration but I wish they just moved over as constant negativism is the worst working environment.   

Best working example of this is Solana. Check here under Economics: https://solana.com/staking#what-will-the-inflation-rate-be 
They lowered rewards but since price is getting higher it doesn't matter to stakers enough to complain at all as value of rewards in USD keeps on getting higher and economics model works. They know higher the issuance lower the price. On the other side are almost all other tokens where price keeps on dropping and rewards are kept the same for stakers and it is an infinite spiral of slow death. Polkadot being one example. 
Point being constant change is needed. Without continued progress and reforms we stay behind. 

I would start with this asap. Keeping it transparent with this discussions here and pinned messages on socials, where every opinion matters and is addressed. Community becomes the best once it is heard and responded too.

**Comment by danairey** [comment]
Personally I think this conversation is irrelevant until Darwinia has an application with users. Users drive demand and darwinia doesnt have users.  Every token is being destroyed other then the main caps as with tokens you essentially own nothing. Users are the only way to create demand. Solana has millions of users so its not a comparison. Polkadot has been a bit of a dud so far and that's the issue , they need users and users in the polkadot system will fall into all the parachains essentially. I dont have an idea how darwinia gets users , the tech deserves it but marketing it with no budget is almost impossible. Only way I see it via some cross chain game where people actually use the tech daily , so to me darwinia should be approaching gaming companies to use the tech. One game can change the whole eco system and bring hundreds of thousands , if not millions to darwinia. The token is going to keep falling as there are no outside buyers, people dont understand the tech and wont understand it, until its applied into a  very useable application that people want to use. That should be the sole focus of darwinia.

**Comment by johneggleston** [comment]
Great idea @hackfisher. 

I am definitely in favour of reducing issuance of RING, unless it’s the issuance of RING bought from the liquidity pools. 

Over the past few months, this has actually been one of the main areas of concern I have been addressing. I have been building an experimental asset on the Arbitrum chain that acts as a deflationary hedge for RING. Basically it’s a token that is heavily paired with RING and there is a 3% tax on every transaction. 1% is burnt, 1% is used to build liquidity, 1% is reflected to holders. The token is also paired with numerous other deflationary tokens on Arbitrum to compound the deflationary pressure.

**Comment by sonoferin2000** [comment]
I support reducing staking rewards, but definitely do not support ceaseing new issuance altogether, want to be clear about that.. if we totally cease new issuance, outright,we basically break the system. it removes all incentive to stake, and would punish people who already staked with the expectation of earning rewards, and unable to remove their stake without incurring penalty. but definitely am on board with the idea of reducing new issuances, and slowing down the rate of creation for kton going forward. .as this would help slow the downward pressure on the token. However, the bigger issue is that there are currently very few practical uses for RING tokens beyond staking, adding them to a liquidity pool, or selling rewards for spending money—actions that only push the price down further. While reducing new emissions is a step in the right direction, it’s not enough on its own.
   We need to create real use cases for the tokens beyond staking and selling, or we won’t see any significant change in the market. Additionally, I believe all DAOs should allocate a portion of their reserves into each other’s tokens as treasury assets. Holding each other’s tokens would benefit both chains by supporting the entire ecosystem, reducing available supply, and helping stabilize liquidity and prices.
   I’m excited about the idea of building treasuries for DAOs, but it's crucial that we keep token holders engaged. One way to do this is by deploying useful decentralized apps (dApps) on both chains, which would drive real usage of the networks.

**Comment by danairey** [comment]
I looked at evolution land and could be so wrong but I personally didn't see that game getting traction. Darwinia does not need to develop the game, we need a gaming company to use the tech for one of their games. A company that specializes purely on making games.  A company that will bring users with them but use our tech for cross chain gaming

**Reply by sonoferin2000** [reply]
the Darwinia project grew out of that game from what i understand, they wanted to connect and interact cross chain, which is really cool origin story.i wanted to play,bought a couple lands on etherium long time ago, spent a fortune in gas opening up drills. breaded a couple villagers, would like to eventually play it, unfortunately those lands are abandoned, im not touching etherium anymore, just too expensive.would like to see it be successful, and would eventually like to actually play.

**Comment by danairey** [comment]
If you really want to get the token going, do something no one in crypto has done. Make the wallet owners of ring and kton the legal owners of the tech. Then go massively public about it, be the first, suddenly there is now real value to the token. That's a game changer and by being the first it brings the tech into the public eye. This whole rubbish of owning tokens which are actually worth zero is coming to an end. Tokens either need real use cases or some sort of intangible value; or they will all trend to 0 . Exactly what is happening.
